### DISCUSSION TITLE
I am getting double underline in the view

### DISCUSSION THREAD

**Discussion by ashutoshBh03** [primefaces/community#1480] [PrimeReact]
when i upgraded the version from 9 to 10 then in tabView ot started showing 2 underlines .
<img width="322" alt="Screenshot 2024-03-20 at 22 34 12" src="https://github.com/primefaces/community/assets/162688961/700b959f-1afe-4b71-b68b-77998f348dcf">

**Comment by melloware** [comment]
Did you also upgrade the theme you are using to match 10.5.3?
